docs/contributors/code/back-merging-to-wp-core.md

```diff
@@ -0,0 +1,33 @@
+# Back Merging code to WordPress Core
+
+For major releases of the WordPress software, Gutenberg features need to be merged into WordPress Core. Typically this involves taking changes made in `.php` files within the Gutenberg repository and making the equivalent updates in the WP Core codebase.
+
+## Files/Directories
+
+Changes to files within the following files/directories will typically require back-merging to WP Core:
+
+-   `lib/`
+-   `phpunit/`
+
+## Ignored directories/files
+
+The following directories/files do _not_ require back-merging to WP Core:
+
+-   `lib`
+    -   `load.php` - Plugin specific code.
+    -   `experiments-page.php` - experiments are Plugin specific.
+-   `packages`
+    -   `block-library` - this is handled automatically during the packages sync process.
+    -   `e2e-tests/plugins` - PHP files related to e2e tests only. Mostly fixture data generators.
+-   ## `phpunit`
```

Comment:
Does this have to be a header?

Reply:
It shouldn't be here at all. Tests often require backports.